public/app/plugins/panel/canvas/CanvasContextMenu.tsx

```diff
@@ -215,6 +215,11 @@ export const CanvasContextMenu = ({ scene, panel }: Props) => {
         currentLayer.doAction(actionType, currentSelectedElement);
       });
     });
+
+    setTimeout(() => {
+      scene.addToSelection();
+      scene.targetsToSelect.clear();
```

Comment:
Not super excited about having this cleanup step here but not sure where else it would make sense to put this :/